CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 2.4.1
+
+- fix: allow code and reason on session close to pass through (#656)
```

Comment:
fix or feature? 🤔

Reply:
it was commited as a fix so I assume it's a fix :D